BlueprintUI/Tests/ViewDiffingBehaviourTests.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  ViewDiffingBehaviourTests.swift
+//  BlueprintUI-Unit-Tests
+//
+//  Created by Kyle Van Essen on 2/20/20.
+//
+
+import Foundation
+import XCTest
+
+@testable import BlueprintUI
+
+class ViewDiffingBehaviourTests : XCTestCase
```

Comment:
Add something to this test that also verifies with applied keys. This is pretty nuanced behaviour, so we should test this.

Reply:
Done